camera_id: 0

numLevels: 5
pyrScale: 0.5
fastPyramids: false
winSize: 13
numIters: 10
polyN: 5
polySigma: 1.1
flags: 0
size_x: 200
size_y: 150
